tests/foreman/cli/test_reporttemplates.py

```diff
@@ -808,3 +818,110 @@ def test_positive_schedule_Entitlements_report(self):
+    @tier3
+    def test_positive_generate_hostpkgcompare(self):
+        """Generate 'Host - compare content hosts packages' report
+
+        :id: 572fb387-86f2-40e2-b2df-e8ec26433610
+
+
+        :setup: Installed Satellite with Organization, Activation key,
+                Content View, Content Host, Subscriptions, and synced fake repo.
+
+        :steps:
+            1. hammer report-template generate --name 'Host - compare content hosts package' ...
+
+        :expectedresults: report is scheduled and generated containing all the expected information
+                          regarding host packages
+
+        :CaseImportance: Medium
+        """
+        # Add subscription to Satellite Tools repo to activation key
+        setup_org_for_a_rh_repo(
+            {
+                'product': PRDS['rhel'],
+                'repository-set': REPOSET['rhst7'],
+                'repository': REPOS['rhst7']['name'],
+                'organization-id': self.setup_org['id'],
+                'content-view-id': self.setup_content_view['id'],
+                'lifecycle-environment-id': self.setup_env['id'],
+                'activationkey-id': self.setup_new_ak['id'],
+            }
+        )
+
+        hosts = []
+        for i in [0, 1]:
+            # Create VM and register content host
+            client = VirtualMachine(distro=DISTRO_RHEL7)
+            client.create()
+            self.addCleanup(vm_cleanup, client)
+            client.install_katello_ca()
+            # Register content host, install katello-agent
+            client.register_contenthost(
+                self.setup_org['label'], self.setup_new_ak['name']
+            )
+            self.assertTrue(client.subscribed)
+            hosts.append(Host.info({'name': client.hostname}))
+            client.enable_repo(REPOS['rhst7']['id'])
+            client.install_katello_agent()
+        hosts.sort(key=lambda host: host['name'])
+
+        host1 = hosts[0]
+        host2 = hosts[1]
+        Host.package_install({'host-id': host1['id'], 'packages': FAKE_0_CUSTOM_PACKAGE_NAME})
+        Host.package_install({'host-id': host1['id'], 'packages': FAKE_1_CUSTOM_PACKAGE})
+        Host.package_install({'host-id': host2['id'], 'packages': FAKE_2_CUSTOM_PACKAGE})
+
+        result = ReportTemplate.generate({'name': 'Host - compare content hosts packages',
+                                         'inputs': f"Host 1 = {host1['name']}, "
+                                          f"Host 2 = {host2['name']}"})
+        result.remove('')
+
+        self.assertGreater(len(result), 1)
+        self.assertEquals(f'Package,{host1["name"]},{host2["name"]},Architecture,Status',
+                          result[0])
+        items = [item.split(',') for item in result[1:]]
+        self.assertGreater(len(items), 0)
+        for item in items:
+            self.assertEquals(len(item), 5)
+            self.assertGreater(len(item[0]), 0)
+            assert((item[1] == '-' and item[0] in item[2])
```

Comment:
instead of item[0], [1], [2], [3], if we use variables name = item[0], it will be little easy to understand next person who will work on the test. This is my personal thought, nonblocking comment